api/views.py

```diff
@@ -138,3 +141,17 @@ def _formatted_bad_request(format_):
     else:
         # json
         return JsonResponse({}, status=400)
+
+@api_view(['GET'])
+def enrichment_view(request):
+    observable_name = request.query_params.get('query')
+    logger.info(f"Enrichment view requested for: {str(observable_name)}")
+    try:
```

Comment:
there is no validation of the input here. This is dangerous with a SQL database. At least we should add some regexes.

Considering that only IP addresses and domains are accepted we could check this 2 regexes (would be even better to merge them into a single one):

* `regex_ip = r"^(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})$"`
* `regex_domain = r"^[a-zA-Z\d-]{1,60}(\.[a-zA-Z\d-]{1,60})*$"`

Reply:
I was not sure what response to return if the given observable does not pass regex checks so returned "Invalid Observable" please resolve this conversation if the recent changes looks satisfactory to you.